packages/docs/developer-resources/integrations/listings.md

```diff
@@ -20,45 +20,52 @@ A general overview of integrations that would be relevant to you listing Celo Pl
 
 ## Resources
-### Address for CELO and cUSD
-The smart contract address for CELO can be found on the explorer [here](https://explorer.celo.org/address/0x471ece3750da237f93b8e339c536989b8978a438/transactions). It is `0x471ece3750da237f93b8e339c536989b8978a438`.
-The smart contract address for cUSD can be found on the explorer [here](https://explorer.celo.org/address/0x765de816845861e75a25fca122bb6898b8b1282a/transactions).
-It is `0x765de816845861e75a25fca122bb6898b8b1282a`.
+### Address for CELO and Stable Value Currencies
+
+* CELO ($CELO) - [`0x471ece3750da237f93b8e339c536989b8978a438`](https://explorer.celo.org/address/0x471ece3750da237f93b8e339c536989b8978a438/transactions)
+* Celo Dollar ($cUSD) - [`0x765de816845861e75a25fca122bb6898b8b1282a`](https://explorer.celo.org/address/0x765de816845861e75a25fca122bb6898b8b1282a/transactions)
+* Celo Euro ($cEUR) - [`0xd8763cba276a3738e6de85b4b3bf5fded6d6ca73`](https://explorer.celo.org/address/0xd8763cba276a3738e6de85b4b3bf5fded6d6ca73/transactions)
 
 
 ### Useful API endpoints
 The following are useful API endpoints available to you that would help you in your listings of the CELO and cUSD digital assets.
 
-#### CELO and cUSD
+#### CELO and Stable Value Currencies
 
 ##### Total CELO supply
 For querying the API on total coins in circulation in CELO, which are the total amount of coins in existence right now, the following endpoint will provide you with that:
 ```sh
 $ curl  [https://thecelo.com/api/v0.1.js?method=ex_totalcoins](https://thecelo.com/api/v0.1.js?method=ex_totalcoins) {"code":"200","msg":"success","data":{"CELO":608485841.9959723,"cUSD":10250632.56099673}}
 ```
 
-##### cUSD Circulating Supply
+##### Stable Value Currencies
+###### cUSD Circulating Supply
 Circulating Supply refers to the # of coins that are circulating in the market and in the general public's hands.
 ```sh
 $ curl https://thecelo.com/api/v0.1.js?method=ex_cusd_circulating
 11353464.550486518
 ```
-
+###### cEUR Circulating Supply
+This endpoint is not yet available.
 #### CP-DOTO (Stability Algorithm)
 
 CP-DOTO information can be found [here](https://docs.celo.org/celo-codebase/protocol/stability/doto).
 
 For API endpoints useful for listing that follow [CMC requirements](https://docs.google.com/document/d/1S4urpzUnO2t7DmS_1dc4EL4tgnnbTObPYXvDeBnukCg/edit#)
 
+##### Mento Addresses
+* cUSD/CELO contract - [`0x67316300f17f063085Ca8bCa4bd3f7a5a3C66275`](https://explorer.celo.org/address/0x67316300f17f063085Ca8bCa4bd3f7a5a3C66275/transactions)
+* cEUR/CELO contract - [`0xE383394B913d7302c49F794C7d3243c429d53D1d`](https://explorer.celo.org/address/0xE383394B913d7302c49F794C7d3243c429d53D1d/transactions)
+
 ##### Summary
-Summary overview of market data for all tickers and all markets.
+Summary overview of market data for all tickers and all markets. This endpoints don't yet support cEUR.
```

Comment:
small grammar fix for plural/singular mismatch:

_if it's a single endpoint (looks like it)_
"This **endpoint doesn't** yet support cEUR."

_multiple endpoints_
"**These** endpoints don't yet support cEUR."

Or, you could reword it and avoid the issue entirely:
"cEUR is not yet supported."